lib/vsc/install/shared_setup.py

```diff
@@ -304,6 +307,33 @@ def reload_vsc_modules(self):
         for name in reload_vsc_modules:
             __import__(name)
 
+    def force_xmlrunner(self):
+        """
+        A monkey-patch attempt to run the tests with
+        xmlrunner.XMLTestRunner(output=xyz).run(suite)
+
+        E.g. in case of jenkins and you want junit compatible reports
+        """
+        import xmlrunner
+        import unittest
```

Comment:
you can just ignore the `ImportError` for `xmlrunner` on top, and complain in here if it's not imported?